iam.tf

```diff
@@ -97,3 +116,9 @@ resource "aws_iam_role_policy" "log_agent" {
   role   = aws_iam_role.task.id
   policy = data.aws_iam_policy_document.task_permissions.json
 }
+
+resource "aws_iam_role_policy" "ecs_exec" {
```

Comment:
will this fail if var.enable_ecs_execute_command = "false" because the policy doc wont exist

Reply:
oops that count was supposed to be removed